h2o-py/h2o/model/model_base.py

```diff
@@ -173,7 +173,7 @@ class c there will be a column Tt.Cc (eg. T3.C1 for tree 3 and class 1). The val
                     data={"predict_staged_proba": True})
         return h2o.get_frame(j["predictions_frame"]["name"])
 
-    def predict_contributions(self, test_data, output_format="Original", top_n=None, top_bottom_n=None, abs_val=False):
+    def predict_contributions(self, test_data, output_format="Original", top_n=None, bottom_n=None, abs=False):
```

Comment:
Hi guys, do you think it is a problem to shadow python abs() function by naming the parameter abs? AFAIK it should have effect only on the current function.

@michalkurka @sebhrusen

Reply:
yes, shadowing in limited scope is usually fine if you think that there's no use for the shadowed function there.
Maybe you could rename to `absolute` otherwise?

Just a suggestion:  I don't know if we're ok to break the client API in a non backwards-compatible way?
For python, you could add a decorator to keep backwards compatibility.
In the same way as we have the `@deprecated_property` decorator, we could create sth like the following:
```
@deprecated_params(dict(
    top_bottom_n=bottom_n,
    abs_val=abs
))
def predict_contributions(self, test_data, output_format="Original", top_n=None, bottom_n=None, abs=False):
```
where the keys of the dictionary are the deprecated names and the values the new ones.
wdyt?

Reply:
> Just a suggestion: I don't know if we're ok to break the client API in a non backwards-compatible way
@sebhrusen this was not released yet, we are still free to change as we find fit

Regarding naming of "abs", I think it would be appropriate to include a verb in the option name. Eg.: "compare_abs" or "abs_compare" - to show what it does. We need to keep in mind that "abs" is only a secondary level parameter - it doesn't affect what predict_contributions return.

Reply:
Regarding the suggestion: I think it is not necessary to have it here. The current API is model.predict_contributions(self, test_data) is working the same was as before

Reply:
I'll make it compare_abs 👍